<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>L2tpPppWidget</class>
 <widget class="QDialog" name="L2tpPppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="grp_authenfication">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblauth">
        <property name="text">
         <string>Allow following authentication methods:</string>
        </property>
        <property name="buddy">
         <cstring>listWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QListWidget" name="listWidget">
        <property name="toolTip">
         <string>Allow/disable authentication methods</string>
        </property>
        <item>
         <property name="text">
          <string>PAP</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHAP</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MSCHAP</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MSCHAPv2</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EAP</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="grp_cryptoAndCompression">
     <property name="title">
      <string>Compression</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbBSD">
        <property name="toolTip">
         <string>Allow/disable BSD-Compress compression.</string>
        </property>
        <property name="text">
         <string>Allow BSD compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbdeflate">
        <property name="toolTip">
         <string>Allow/disable Deflate compression.</string>
        </property>
        <property name="text">
         <string>Allow Deflate compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbTCPheaders">
        <property name="toolTip">
         <string>Allow/disable Van Jacobson style TCP/IP header compression in both the transmit
and the receive directions.</string>
        </property>
        <property name="text">
         <string>Allow TCP header compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbCompressionNegotiation">
        <property name="toolTip">
         <string>Allow protocol field compression negotiation in both the receive and the
 transmit directions.</string>
        </property>
        <property name="text">
         <string>Use protocol field compression negotiation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbAddressControlCompression">
        <property name="toolTip">
         <string>Use Address/Control compression in both directions (send and receive).</string>
        </property>
        <property name="text">
         <string>Use Address/Control compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="grp_echo">
     <property name="title">
      <string>Echo</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbsendEcho">
        <property name="toolTip">
         <string>Send LCP echo-requests to find out whether peer is alive.</string>
        </property>
        <property name="text">
         <string>Send PPP echo packets</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QGroupBox" name="grp_other">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Other Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbMRU">
        <property name="text">
         <string>MRU: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbMRU">
        <property name="maximum">
         <number>1500</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbMTU">
        <property name="text">
         <string>MTU: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbMTU">
        <property name="maximum">
         <number>1500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbMPPE">
     <property name="toolTip">
      <string>Note: MPPE encryption is only available with MSCHAP authentication methods. To
enable this checkbox, select one or more of the MSCHAP authentication
methods: MSCHAP or MSCHAPv2.</string>
     </property>
     <property name="title">
      <string>Use MPPE Encryption</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMMPEcrypto">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Crypto:</string>
        </property>
        <property name="buddy">
         <cstring>cbMPPECrypto</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbMPPECrypto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Require the use of MPPE, with 40/128-bit encryption or all.</string>
        </property>
        <item>
         <property name="text">
          <string comment="like in use Any configuration">Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40 bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="cbstatefulEncryption">
        <property name="toolTip">
         <string>Allow MPPE to use stateful mode. Stateless mode is still attempted first.</string>
        </property>
        <property name="text">
         <string>Use stateful encryption</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>grp_authenfication</zorder>
  <zorder>grp_echo</zorder>
  <zorder>grp_cryptoAndCompression</zorder>
  <zorder>gbMPPE</zorder>
  <zorder>buttonBox</zorder>
  <zorder>grp_other</zorder>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>gbMPPE</tabstop>
  <tabstop>cbMPPECrypto</tabstop>
  <tabstop>cbstatefulEncryption</tabstop>
  <tabstop>cbBSD</tabstop>
  <tabstop>cbdeflate</tabstop>
  <tabstop>cbTCPheaders</tabstop>
  <tabstop>cbCompressionNegotiation</tabstop>
  <tabstop>cbAddressControlCompression</tabstop>
  <tabstop>cbsendEcho</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>L2tpPppWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>L2tpPppWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbMPPE</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbMPPECrypto</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gbMPPE</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbstatefulEncryption</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
